### PR TITLE
Add hacks from BHNT96

### PR DESCRIPTION
**Pull request by ligi**

### CODE REVIEW

**Comment by ligi** [comment]
cc @hartwork @georgjaehnig @pkoryzna @wleuschner @kevincharm  @ashgansh @timstr


- [trovu.net](https://github.com/trovu/trovu) by [Georg](https://github.com/georgjaehnig)
- [AI File Oranizer 2000](https://github.com/different-ai/file-organizer-2000) by [ben](https://github.com/ashgansh)
- [rageagainst.eu](https://github.com/kevincharm/rageagainst) by serg
- eink_display_todos by [patryk](https://github.com/pkoryzna)
- [OceanWaves](https://github.com/wleuschner/TessendorfWaves) by [vlad](github.com/wleuschner)
- Geiger Counter Stuff by [Tim](https://timstr.website) - **Hack of the Month**

**Comment by timstr** [comment]
Hi @ligi thanks for all the fun last night! Would you mind sending me the trophy photo at straubinger.tim@gmail.com ?

File: _posts/2024-05-28-no96-portal-hacks.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ location: '[c-base](https://www.c-base.org)'
...
 
-Upcoming - please [RSVP here](https://www.meetup.com/de-DE/berlin-hack-and-tell/events/301152612)
-
+- [binary-gentoo](https://github.com/hartwork/binary-gentoo) by [hartwork](https://github.com/hartwork)
+- Letters of Intent by tinabel
 - [trovu.net](https://github.com/trovu/trovu) by [Georg](https://github.com/georgjaehnig)
+- [AI File Oranizer 2000](https://github.com/different-ai/file-organizer-2000) by [ben](https://github.com/ashgansh)
+- [rageagainst.eu](https://github.com/kevincharm/rageagainst) by serg
+- eink_display_todos by [patryk](https://github.com/pkoryzna)
+- [OceanWaves](https://github.com/wleuschner/TessendorfWaves) by [vlad](github.com/wleuschner)
```
Comment (hartwork) [inline _posts/2024-05-28-no96-portal-hacks.md:17]:
```diff
--- github.com/wleuschner
+++ https://github.com/wleuschner
```

```diff
--- vlad
+++ Wladimir
```
(or "wlad" with "w"?)

CC @wleuschner

**Review by hartwork — COMMENTED** [review]
One broken link, one potentially misspelled nick? Please see below.

---
PS: Be sure to check out @timstr's linked [website](https://timstr.website/), e.g. I found this gem in the [graphics section](https://timstr.website/graphics/pathtracing/):


Have a nice day!

File: _posts/2024-05-28-no96-portal-hacks.md (hunk 1 above)
Comment (wleuschner) [reply _posts/2024-05-28-no96-portal-hacks.md:17]: vlad is the right nick in this context

**Comment by timstr** [comment]
> Hi @ligi thanks for all the fun last night! Would you mind sending me the trophy photo at [straubinger.tim@gmail.com](mailto:straubinger.tim@gmail.com) ?

@ligi bump

**Comment by ligi** [comment]
here you find the trophy pic: https://x.com/mr_ligi/status/1795887726199255122

**Pull request merged**